Chad dev

halvorsenca opened the pull request:
this is the list counter method please let me know if I am fucking stupid or if it looks good

Commit:
finished list counter

Commit:
Merge branch 'master' of https://github.com/JaysGitLab/cs-5666-scatt-brandy.git into ChadDev

matts-dev commented:
I wrote a test for it, it doesn't pass it. :\ It works fine, except for when they're multiple locations lists pop up (which was the case for the file I am using). I'm going to go ahead and merge it so we have a place to start to fix it.

matts-dev commented:
Also, there isn't anything in student using the list count.

matts-dev committed:
Added some more data to test that contains 13 lists and 13 variables
(among other things). Also wrote a test for the list counter, currently
it isn't passing but since there now exists a test I want to go ahead
and commit that and merge it to the master.

matts-dev reviewed (commented):
I'll add it to master, but needs some work on lists that are separated into different locations within the json.
